integration-tests/src/test/java/oracle/kubernetes/operator/ITSessionMigration.java

```diff
@@ -0,0 +1,393 @@
+// Copyright 2018, Oracle Corporation and/or its affiliates.  All rights reserved.
+// Licensed under the Universal Permissive License v 1.0 as shown at
+// http://oss.oracle.com/licenses/upl.
+
+package oracle.kubernetes.operator;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import oracle.kubernetes.operator.utils.Domain;
+import oracle.kubernetes.operator.utils.ExecCommand;
+import oracle.kubernetes.operator.utils.ExecResult;
+import oracle.kubernetes.operator.utils.Operator;
+import oracle.kubernetes.operator.utils.TestUtils;
+import org.junit.AfterClass;
+import org.junit.Assume;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+/**
+ * Simple JUnit test file used for testing Operator.
+ *
+ * <p>This test is used for creating Operator(s) and domain(s) which are managed by the Operator(s).
+ * And to test WLS Session Migration feature
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class ITSessionMigration extends BaseTest {
+
+  // property file used to customize operator properties for operator inputs yaml
+  private static String operatorYmlFile = "operator1.yaml";
+
+  // file used to customize domain properties for domain, PV and LB inputs yaml
+  private static String domainonpvwlstFile = "domainonpvwlst.yaml";
+
+  // property file used to configure constants for integration tests
+  private static String appPropsFile = "OperatorIT.properties";
+
+  private static final String TESTAPPNAME = "httpsessionreplication";
+  private static Map<String, String> httpAttrMap;
+
+  private static String httpHeaderFile;
+
+  private static Operator operator;
+  private static Domain domain;
+
+  private static boolean QUICKTEST;
```

Comment:
This code is not needed here, all the common code is moved to BaseTest. Take a look at other IT classes.

Reply:
Thanks. The change is on the way